otm/resources/schemas/otm_schema.json

```diff
@@ -49,18 +49,18 @@
             "properties": {
                 "name": {"type": "string"},
                 "id": {"type": "string"},
-                "description": {"type": "string"},
+                "description": {"type": ["string", "null"]},
                 "risk": {
-                    "type": "object",
+                    "type": ["object", "null"],
```

Comment:
"risk" is a required field so it shouldn't be null, instead "assets" is optional si a null should be allowed (line 47)